q3.js

```diff
@@ -0,0 +1,46 @@
+let obj1 = {
+    "channel": "A",
+    "name": "shoe"
+  };
+let obj2 = {
+    "channel": "A",
+    "name": "electronics"
+  };
+let obj3 = {
+    "channel": "B",
+    "name": "apparel"
+  };
+let obj4 = {
+    "channel": "C",
+    "name": "electronics"
+  };
+
+
+let arrayList = new Array();
+arrayList.push(obj1);
+arrayList.push(obj2);
+arrayList.push(obj3);
+arrayList.push(obj4);
+
+console.log(arrayList);
+
+let ans = function groupObjectsBy(jsonObjects,key){
```

Comment:
you should use either of the below 2 syntax for creating a function:
A.   let groupObjectsBy = function () {}
B.   function groupObjectsBy() {}

You have mixed the both syntaxes.

Reply:
Made the changed